src/api/app/assets/javascripts/webui2/buildresult.js

```diff
@@ -0,0 +1,66 @@
+function buildresultTabs() {
```

Comment:
This really doesn't make too much sense. It also breaks the tab. 

![screenshot from 2018-08-29 16-25-03](https://user-images.githubusercontent.com/514785/44794136-1c276780-aba8-11e8-90c4-aba8deb2bed1.png)

I would suggest you move the functionality into `.tab-content`...